docs/apis/http/tus_upload.md

```diff
@@ -191,3 +220,29 @@ curl -ks -XPOST https://ocis.test/remote.php/dav/spaces/{space-id} \
+| `currentFolder`                  | /                                                                                                                                               | Sent by the web UI                                                                                  |
+| `currentFolderId`                | 8748cddf-66b7-4b85-91a7-e6d08d8e1639$a9778d63-21e7-4d92-9b47-1b81144b9993!a9778d63-21e7-4d92-9b47-1b81144b9993                                  | Sent by the web UI                                                                                  |
+| `uppyId`                         | uppy-example/pdf-1e-application/pdf-238300                                                                                                      | Sent by the web UI                                                                                  |
+| `relativeFolder`                 |                                                                                                                                                 | File path relative to the folder that is being uploaded, without filename. Sent by the web UI.      |
+| `tusEndpoint`                    | https://ocis.ocis-traefik.latest.owncloud.works/remote.php/dav/spaces/8748cddf-66b7-4b85-91a7-e6d08d8e1639$a9778d63-21e7-4d92-9b47-1b81144b9993 | Sent by the web UI                                                                                  |
+| `uploadId`                       | 71d5f878-a96c-4d7b-9627-658d782c93d7                                                                                                            | Sent by the web UI                                                                                  |
+| `topLevelFolderId`               | undefined                                                                                                                                       | Sent by the web UI                                                                                  |
+| `routeName`                      | files-spaces-generic                                                                                                                            | Sent by the web UI                                                                                  |
+| `routeDriveAliasAndItem`         | cGVyc29uYWwvYWRtaW4=                                                                                                                            | Sent by the web UI                                                                                  |
+| `routeShareId`                   |                                                                                                                                                 | Share ID when uploading into a received folder share. Sent by the web UI                            |
```

Comment:
Actually the whole "Sent by the web UI" comments should also explain what the items means, and why they are sent.

Reply:
@JammingBen could you elaborate or remove them if you think they're not relevant.

This is @JammingBen 's previous statement:

> Web needs this information so it can correctly display and resole into uploaded files and folders after a finished upload. The easiest way to achieve this was to simply put all that information into metadata because Web can still retrieve that information after uploading.
> 
> I don't know if the server is doing anything with all that information, my guess would be no. Also, other clients probably don't need all that stuff.